c2corg_ui/format/ltag.py

```diff
@@ -0,0 +1,64 @@
+"""
+LTag Extension for Python-Markdown
+====================================
+
+See https://regex101.com/r/g2Zjqn/2/ for regex tests
+
+"""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from markdown.extensions import Extension
+from markdown.blockprocessors import BlockProcessor
+from markdown.util import etree
+import re
+
+
+class LTagProcessor(BlockProcessor):
+    """ Process Tables. """
+
+    RE_LTAG_BLOCK = re.compile(r'^(?:L#(?:[^|\n]*(?:\|[^|\n]*)+)\n?)+$')
+
+    def __init__(self, parser):
+        super(LTagProcessor, self).__init__(parser)
+
+    def test(self, parent, block):
+        is_table = self.RE_LTAG_BLOCK.search(block) is not None
+
+        return is_table
```

Comment:
Simplify yes, but the name is imposed by the Markdown Extension API : 
> The test method is used by BlockParser to identify the type of block. Therefore the test method must return a Boolean value. If the test returns True, then the BlockParser will call that Blockprocessor’s run method. If it returns False, the BlockParser will move on to the next Blockprocessor.
https://pythonhosted.org/Markdown/extensions/api.html#blockparser

After re-reading the documentation, I've understood that I don't need to call the test method in the run method :)